src/full/Agda/Syntax/Parser/Literate.hs

```diff
@@ -134,11 +134,6 @@ bleach s = map go s
 isBlank :: Char -> Bool
 isBlank = (&&) <$> isSpace <*> not . (== '\n')
 
--- | Possible extensions for a literate Agda file.
-
-literateExts :: [String]
-literateExts = map fst literateProcessors
```

Comment:
Used only once, imported many times.
This looks an unnecessary function to me, I just inlined the only usage and removed all the other imports.